### DISCUSSION TITLE
How to query jsonB entries?

### DISCUSSION THREAD

**Discussion by Flajt** [supabase/supabase#4278] [Questions] [answered]
Hello everybody,
I would like to know how one can use jsonB values for querying (basic CRUD). With the supabase libraries (for example dart). As there seems no way to use the `->` operator, apart from:
```dart
from("mytable").select("mycolumn->myJsonKey")
``` 
but this is not filterable or allows the user to access any other data fields in the jsonB.
The only idea we have got is to do all the filtering on the client.

Any help/advice is appreciated, we might have missed it as we are newbies to SQL so sorry in advance.

**Comment by soedirgo** [comment, marked as answer]
Hey @Flajt, you can use the `->` operator with filters: https://supabase.com/docs/reference/javascript/select#querying-json-data

> or allows the user to access any other data fields in the jsonB

Can you elaborate on this? I'm assuming that the user is allowed to see everything in `mycolumn`.
